### PR TITLE
add new demo for sensu+kafka

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**

### CODE REVIEW

**Comment by Alex-Izquierdo** [comment]
please review @benthomasson @bzwei

File: demos/sensu-kafka/instructions.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Ansible Events + sensu + kafka DEMO
```
Comment (mkanoor) [inline demos/sensu-kafka/instructions.md:1]: @Alex-Izquierdo Can you put a picture of all the different components

Comment (mkanoor) [reply demos/sensu-kafka/instructions.md:1]: @Alex-Izquierdo Any help for the developers who have minikube?

Comment (Alex-Izquierdo) [reply demos/sensu-kafka/instructions.md:1]: @mkanoor added diagram.

Comment (Alex-Izquierdo) [reply demos/sensu-kafka/instructions.md:1]: @mkanoor I'm not sure if providing multiple environments is in the scope of this task or if it's the priority. The current approach is based on a widely adopted technology that can work straightforwardly in the main operative systems covering the vast majority of local environments. 

In fact, you can [install docker-compose as a standalone binary](https://docs.docker.com/compose/install/compose-plugin/#install-the-plugin-manually) in your minikube VM and execute the demo there. 
```
sudo curl -SL https://github.com/docker/compose/releases/download/v2.7.0/docker-compose-linux-x86_64 -o /usr/local/bin/docker-compose
```

**Review by benthomasson — APPROVED** [review]

File: demos/sensu-kafka/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      sensuctl asset add sensu/http-checks &&
+      sensuctl asset add sensu/sensu-kafka-handler &&
+      sensuctl create -f /sensu-assets/check.yml &&
+      sensuctl create -f /sensu-assets/handler.yml"
+    depends_on:
+      sensu-backend:
+        condition: service_healthy
+
+  ansible-events:
+    image: quay.io/aizquier/ansible-events-dev:latest
```
Comment (bzwei) [inline demos/sensu-kafka/docker-compose.yml:103]: @Alex-Izquierdo Is benthomasson-eda collection already installed in the image?

**Comment by bzwei** [comment]
LGTM

File: demos/sensu-kafka/docker-compose.yml (hunk 2 above)
Comment (Alex-Izquierdo) [reply demos/sensu-kafka/docker-compose.yml:103]: @bzwei Of course, it wouldn't work otherwise.

**Comment by mkanoor** [comment]
@Alex-Izquierdo Can you paste a copy of the event that flows from Sensu to Kafka does it have a unique id or a time?

**Comment by Alex-Izquierdo** [comment]
> @Alex-Izquierdo Can you paste a copy of the event that flows from Sensu to Kafka does it have a unique id or a time?

@mkanoor can you expand on the question? I'm not sure what exactly you want. The event has indeed an UUID and epoch timestamp:

```py
{
    "check": {
        "command": "http-check --url http://fake-app:5080/health",
        "handlers": ["sensu-kafka-handler"],
        "high_flap_threshold": 0,
        "interval": 5,
        "low_flap_threshold": 0,
        "publish": True,
        "runtime_assets": ["sensu/http-checks"],
        "subscriptions": ["webserver"],
        "proxy_entity_name": "",
        "check_hooks": None,
        "stdin": False,
        "subdue": None,
        "ttl": 0,
        "timeout": 0,
        "round_robin": False,
        "duration": 0.008274141,
        "executed": 1660040476,
        "history": [
            {"status": 0, "executed": 1660040376},
            {"status": 0, "executed": 1660040381},
            {"status": 0, "executed": 1660040386},
            {"status": 0, "executed": 1660040391},
            {"status": 0, "executed": 1660040396},
            {"status": 0, "executed": 1660040401},
            {"status": 0, "executed": 1660040406},
            {"status": 0, "executed": 1660040411},
            {"status": 0, "executed": 1660040416},
            {"status": 0, "executed": 1660040421},
            {"status": 0, "executed": 1660040426},
            {"status": 0, "executed": 1660040431},
            {"status": 0, "executed": 1660040436},
            {"status": 0, "executed": 1660040441},
            {"status": 0, "executed": 1660040446},
            {"status": 0, "executed": 1660040451},
            {"status": 0, "executed": 1660040456},
            {"status": 0, "executed": 1660040461},
            {"status": 0, "executed": 1660040466},
            {"status": 0, "executed": 1660040471},
            {"status": 0, "executed": 1660040476},
        ],
        "issued": 1660040476,
        "output": "http-check OK: HTTP Status 200 for http://fake-app:5080/health\n",
        "state": "passing",
        "status": 0,
        "total_state_change": 0,
        "last_ok": 1660040476,
        "occurrences": 56,
        "occurrences_watermark": 56,
        "output_metric_format": "",
        "output_metric_handlers": None,
        "env_vars": None,
        "metadata": {
            "name": "check-fake-app",
            "namespace": "default",
            "labels": {"sensu.io/managed_by": "sensuctl"},
        },
        "secrets": None,
        "is_silenced": False,
    },
    "entity": {
        "entity_class": "agent",
        "system": {
            "hostname": "ba931177f487",
            "os": "linux",
            "platform": "alpine",
            "platform_family": "alpine",
            "platform_version": "3.12.12",
            "network": {
                "interfaces": [
                    {"name": "lo", "addresses": ["127.0.0.1/8", "::1/128"]},
                    {
                        "name": "eth0",
                        "mac": "46:66:53:84:4f:a0",
                        "addresses": ["10.89.0.52/24", "fe80::4466:53ff:fe84:4fa0/64"],
                    },
                ]
            },
            "arch": "amd64",
            "libc_type": "musl",
            "vm_system": "kvm",
            "vm_role": "host",
            "cloud_provider": "",
            "processes": None,
        },
        "subscriptions": ["webserver", "entity:ba931177f487"],
        "last_seen": 1660040476,
        "deregister": False,
        "deregistration": {},
        "user": "admin",
        "redact": [
            "password",
            "passwd",
            "pass",
            "api_key",
            "api_token",
            "access_key",
            "secret_key",
            "private_key",
            "secret",
        ],
        "metadata": {"name": "ba931177f487", "namespace": "default"},
        "sensu_agent_version": "6.7.4",
    },
    "id": "951bb8cd-9ac0-46be-a29c-bd8f0765ff1a",
    "metadata": {"namespace": "default"},
    "timestamp": 1660040476,
}
```

**Review by benthomasson — APPROVED** [review]

**Pull request merged**